cli/cmd/plugin/unmanaged-cluster/semver/semver.go

```diff
@@ -0,0 +1,317 @@
+//nolint
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Question: do we also include our license header here? I put both, but i don't know the convention. Maybe @stmcginnis knows?

Reply:
Sounds good! That's what I've done and looks like the linters are happy!